src/helpers/drive-link-handler.ts

```diff
@@ -0,0 +1,264 @@
+import { Context } from "../types";
+import { bubbleUpErrorComment } from "./errors";
+
+const POLL_INTERVAL = 25000; // 25 seconds
+const MAX_POLL_TIME = 900000; // 15 minutes
```

Comment:
Why 25 seconds and 15 minutes? I think they are reasonable defaults but curious for rationale.

Reply:
We’ll be using two APIs, Drive (with a rate limit of 600 RPM or 1000 Request per 100 seconds) and Sheets (with a rate limit of 300 RPM). To simplify the polling logic, I want to ensure the combined rate does not exceed 2 RPM, even when scaled, providing a comfortable buffer. This approach can easily be adjusted to intervals of 5 or 10 seconds as needed.